hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestCustomerProvidedKey.java

```diff
@@ -106,7 +106,7 @@ public ITestCustomerProvidedKey() throws Exception {
   @Test
   public void testReadWithCPK() throws Exception {
     final AzureBlobFileSystem fs = getAbfs(true);
-    String fileName = path("/" + methodName.getMethodName()).toString();
+    String fileName = "/" + methodName.getMethodName();
```

Comment:
so this moves away from path(). why?

Reply:
This is an older PR, will be abandoning this. The current PR is at #3153
Could you please review that instead?